bin/docker-image-tool.sh

```diff
@@ -67,6 +67,16 @@ function docker_push {
   fi
 }
 
+function resolve_file {
+  local FILE=$1
+  if [ -n "$FILE" ]; then
+    local DIR=$(dirname $FILE)
```

Comment:
This block is basically `readlink -f`.

Reply:
Which doesn't work on OS X/BSD where `-f` is a format specifier not follow links

Also the intent was not to attempt to resolve symbolic links simply to make the path absolute so the `cd` into the build context directory doesn't render the relative path invalid